README.md

```
@@ -15,7 +15,7 @@
     <img src="https://img.shields.io/badge/Donate-Patreon-F96854.svg?logo=patreon" alt="Support Us">
   </a>
   <a href="https://www.python.org/downloads/">
-    <img src="https://img.shields.io/badge/Made%20With-Python%203.10-blue.svg?style=for-the-badge" alt="Made with python">
+    <img src="https://img.shields.io/badge/Made%20With-Python%203.13-blue.svg?style=for-the-badge" alt="Made with python">
```

Markdownlint (0.35.0)

Element: img
Inline HTML

(MD033, no-inline-html)